src/cpp/src/whisper_generation_config.cpp

```diff
@@ -38,6 +42,8 @@ WhisperGenerationConfig::WhisperGenerationConfig(const std::filesystem::path& js
     }
 
     read_json_param(data, "lang_to_id", lang_to_id);
+
+    apply_chat_template = false;
```

Comment:
Use member initializer list for both of the constructors

Reply:
it fails on compilation with: error: `class ov::genai::WhisperGenerationConfig does not have any field named apply_chat_template`

Reply:
Yes, my bad